app/models/events/new_discussion.rb

```diff
@@ -7,7 +7,12 @@ def self.publish!(discussion)
     create(kind: 'new_discussion',
            user: discussion.author,
            announcement: discussion.make_announcement,
-           eventable: discussion).tap { |e| EventBus.broadcast('new_discussion_event', e) }
+           eventable: discussion,
+           created_at: discussion.created_at).tap { |e| EventBus.broadcast('new_discussion_event', e) }
```

Comment:
We need to ensure that the `created_at` values for the events match correctly, otherwise we might end up with off-by-millisecond errors when 'reading' content that you just created.